solutions/standard/variables.tf

```diff
@@ -122,6 +122,24 @@ variable "watsonx_ai_runtime_service_endpoints" {
   }
 }
 
+variable "watsonx_ai_new_project_members" {
+  description = "The list of new members the owner of the Watsonx.ai project would like to add to the project."
+  type = list(object({
+    email  = string
+    iam_id = string
+    role   = string
+    })
+  )
+  default = []
+
+  validation {
+    condition = alltrue([
+      for member in var.watsonx_ai_new_project_members : contains(["admin", "editor", "viewer"], member.role)
+    ])
+    error_message = "The specified new member role is not valid. Supported options are admin, editor, or viewer."
```

Comment:
Since this is a complex object type, you need to add a supporting markdown doc which explains the required format since people will be consuming this in projects UI, and would not know what is required format. This variable description should point to the doc. For reference, see https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/fbfe71b4705e422018300bdecd37d5f99f63f244/solutions/standard/variables.tf#L115